PocketKit/Sources/PocketKit/MyList/CompactMyListContainerCoordinator.swift

```diff
@@ -166,17 +170,21 @@ class CompactMyListContainerCoordinator: NSObject {
         let readerSettingsVC = ReaderSettingsViewController(settings: readable.readerSettings) { [weak self] in
             self?.model.clearIsPresentingReaderSettings()
         }
+        readerSettingsVC.configurePocketDefaultDetents()
+        viewController.present(readerSettingsVC, animated: !isResetting)
+    }
 
-        // iPhone (Portrait): defaults to .medium(); iPhone (Landscape): defaults to .large()
-        // By setting `prefersEdgeAttachedInCompactHeight` and `widthFollowsPreferredContentSizeWhenEdgeAttached`,
-        // landscape (iPhone) provides a non-fullscreen view that is dismissable by the user.
-        let detents: [UISheetPresentationController.Detent] = [.medium(), .large()]
-        readerSettingsVC.sheetPresentationController?.detents = detents
-        readerSettingsVC.sheetPresentationController?.prefersGrabberVisible = true
-        readerSettingsVC.sheetPresentationController?.prefersEdgeAttachedInCompactHeight = true
-        readerSettingsVC.sheetPresentationController?.widthFollowsPreferredContentSizeWhenEdgeAttached = true
+    func presentSortMenu(presentedSortFilterViewModel: SortMenuViewModel?) {
+        guard let sortFilterVM = presentedSortFilterViewModel else {
```

Comment:
A note on coordinators and presenting modal content:

When users transition from `compact` to `regular` size classes, we will actually rebuild the entire UI hierarchy _with the exception of any modal views_. Because of this, we need to avoid presenting modal content more than once.

Do prevent this code from presenting a modal more than once, add `guard !isResetting else { return }` to the top of the function body.